go/scanner/main/scanner.go

```diff
@@ -96,7 +97,7 @@ func createMatcherFromFlags() (scanner.Matcher, error) {
 
 func main() {
 	flag.Parse()
-	logClient := client.New(*logUri)
+	logClient := client.New(*logUri, &http.Client{})
```

Comment:
Here too, set up the transport? I think this particular call site is why we came to use that helper library, IIRC...